Mautic 3.0 documentation changes (#64)
mautic/mautic-documentation: pages/22.What's New/docs.en.md

----------------------------
In this section we'll highlight the new features for each major release of Mautic.  They are in version order beginning with the latest.

> ##Version 3.0

Reviewer: As applications [don't always handle a missing space correctly](https://www.markdownguide.org/basic-syntax/#heading-best-practices), let's make sure we leave one just in case.

> Mautic 3.0 introduces significant features and a number of enhancements. They are outlined as follows:

Reviewer: This release does not include any features or enhancements, so this might be a bit misleading.

```suggestion
Mautic 3.0 introduces significant changes to the underlying application along with some changes to the minimum version requirements to run Mautic. They are outlined as follows:
```

###Hosting Requirements

> - Mautic 3.0 added support for PHP versions 7.2 and 7.3. The minimum PHP version is raised from version 5.6.19 to version 7.2.0.

Reviewer: Support for 7.2 and 7.3 were available in 2.16 releases so I don't think we need to include that here. 

```suggestion
- Mautic 3.0 raised the minimum PHP version required from 5.6.19 to 7.2.0.
```

- The minimum MySQL version is increased from version 5.5.3 to 5.7.14.

###Cron Jobs Update
There are significant changes in the file structure in Mautic 3. To comply with the change, you will need to update your cron jobs path.

Example:

`app/console mautic:segments:update`

should now be:

`bin/console mautic:segments:update`

> Note: For Mautic version 2.0, you must continue using the `app/console` path for running any cron job.

###Removed Plugins
Mautic 3.0 removed support for the Rackspace and OpenStack plugins for remote assets.

> While uploading an asset on the New Asset page, if you select Remote Storage location and click Browse Remote Files, you will see only Amazon S3 as the available plugin. The Rackspace and OpenStack plugins are no longer available.

Reviewer: They will only see these options if they have them configured, so I would perhaps suggest removing this as I don't think it would be likely for people to have all three plugins enabled and configured.  Generally people will use one storage provider in a Mautic instance.

```suggestion
If you currently use Rackspace or OpenStack you will need to switch to Amazon S3, or create custom plugins to support alternative providers.
```

> Note: For Mautic version 2.0, you must continue using the `app/console` path for running any cron job.

### Removed Plugins
Mautic 3.0 removed support for the Rackspace and OpenStack plugins for remote assets.

If you currently use Rackspace or OpenStack, you will need to switch to Amazon S3, or create custom plugins to support alternative providers.

### New File Manager for WYSIWYG Editors
When you try to upload an image or replace an existing image using the email or page builder, you will see a new file manager which you can use to drag and drop images.

![The new file manager window](file-manager.png)

> 

Reviewer: Great!

###Froala editor

> This release switches CKEditor in favor of Froala editor which has a more polished look and functionality.

Reviewer: We are not ditching Froala in Mautic 3.0 - Perhaps you mean the file manager? In that case please see the comments on the slack thread.

Reviewer: Sorry, I completely misread and thought this was in the 3.x part! :blush:
